docs/data-modeling/field-types.md

```diff
@@ -10,33 +10,35 @@ While data types indicate to a database how it should interpret the values in a
 
 Field types dictate how Metabase displays its data, as well as the column’s special functionality, if any. By marking columns in a table as **Latitude** and **Longitude**, Metabase can use the columns to create pin and heat maps. Similarly, designating a column as a **URL** allows users to click on the link to visit that URL.
 
-## List of Metabase Field Types
+## Overall Row
 
-Metabase recognizes the following field types:
+- **Entity Key**: The field in this table that uniquely identifies each row. Could be a product ID, serial number, etc.
+- **Entity Name**: Different from the entity key, the entity name represents what each row in the table _is_. For example, in a Users table, the User column might be the entity name.
```

Comment:
Re: `the User column might be the entity name.`
I have trouble understanding what is meant by that. What would be in a User column? Was this supposed to be `username` ?